pyproject.toml

```diff
@@ -1,3 +1,143 @@
+[tool.poetry]
+name = "ibis-framework"
+version = "1.4.0"
+packages = [{ include = "ibis" }]
+homepage = "https://ibis-project.org"
+repository = "https://github.com/ibis-project/ibis"
+documentation = "https://ibis-project.org/docs"
+description = "Productivity-centric Python Big Data Framework"
+authors = ["Ibis Contributors"]
+maintainers = ["Ibis Contributors"]
+license = "Apache-2.0"
+readme = "README.md"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Operating System :: OS Independent",
+  "Intended Audience :: Science/Research",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Topic :: Scientific/Engineering",
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7.1,<4"
+cached_property = "^1"
+multipledispatch = "^0.6"
+numpy = "^1"
+pandas = "^1"
+pytz = "2021.1"
+regex = "2021.7.6"
+cytoolz = "^0.11"
+toolz = "^0.11"
+clickhouse-driver = { version = "^0.1", optional = true }
+clickhouse-sqlalchemy = { version = "^0.1", optional = true }
+dask = { version = ">2021.6.0", optional = true, extras = [
+  "dataframe",
+  "array"
+] }
+geoalchemy2 = { version = "^0.9", optional = true }
+geopandas = { version = "^0.9", optional = true }
+graphviz = { version = "^0.16", optional = true }
+hdfs = { version = "^2", optional = true }
+importlib-metadata = { version = "^4", python = "<3.8" }
+impyla = { version = "0.18a1", optional = true, extras = ["kerberos"] }
+psycopg2 = { version = "^2", optional = true }
+pyarrow = { version = "^5", optional = true }
```

Comment:
is this bumping the min pyarrow version?

Reply:
It is. I can lower it if desired.

Reply:
This now set to 1.0